### PR TITLE
ConfirmPasswordResetForm: error message now showing

### PR DESCRIPTION
**Pull request by greeffjean**
The issue was the "response" callback parameter in our .then() chain was not being read in a if/else statement inside the actual .then() chain.

Test for this bug fix to follow

### CODE REVIEW

**Comment by greeffjean** [comment]
Hi guys. 

This will the last and final fix for this bug I believe as the issue was a mere spelling error. As mentioned before test for this fix to follow as soon as I get setup with Cypress, Kaleab is currently assisting me to get up and running.

File: docker-compose.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ redis:
 db:
   image: orchardup/postgresql
   ports:
-    - 5432:5432
+    - 4040:5432
```
Comment (avishkar58) [inline docker-compose.yml:8]: Please remove this change from the PR so we don't change the port for everyone else who already has things set up

File: webapp/package.json [hunk 2]
```diff
@@ -9,16 +9,16 @@
...
     "moment": "2.24.0",
     "query-string": "^6.5.0",
-    "react": "^16.7.0",
+    "react": "^16.13.1",
     "react-app-polyfill": "^0.2.2",
     "react-bootstrap4-modal": "^1.7.0",
     "react-cookie-consent": "^3.0.0",
     "react-datetime-picker": "^2.9.0",
-    "react-dom": "^16.7.0",
+    "react-dom": "^16.13.1",
```
Comment (avishkar58) [inline webapp/package.json:21]: Awesome thanks for doing these upgrades! :)

File: webapp/src/pages/resetPassword/components/RequestPasswordResetForm.js [hunk 3]
```diff
@@ -40,11 +40,13 @@ class RequestPasswordResetForm extends Component {
...
           });
         } else {
+
           this.setState({
-            error: response.message,
+            error: response.message, 
             loading: false
           });
         }
+        console.log(response)
```
Comment (avishkar58) [inline webapp/src/pages/resetPassword/components/RequestPasswordResetForm.js:49]: Please remove debugging logs

**Comment by greeffjean** [comment]
Done

**Comment by avishkar58** [comment]
Awesome thanks @greeffjean ! Your first PR merged into Baobab :)

**Pull request merged**